packages/now-cli/src/util/index.js

```diff
@@ -145,6 +145,7 @@ export default class Now extends EventEmitter {
       regions,
       target: target || undefined,
       projectSettings,
+      source: 'cli',
```

Comment:
This information is already in the the `User-Agent` header. I don't think we need a new property to detect the CLI.

Reply:
This is because user-agent is not enough for other cases. I think it's better to not have special cases for handling CLI requests on API, not strong opinion tho.

Reply:
Looks good now that the API is updated 👍